code/game/objects/items/devices/radio/radio.dm

```diff
@@ -186,6 +186,20 @@
 				. = TRUE
 
 /obj/item/radio/talk_into(atom/movable/M, message, channel, list/spans, datum/language/language, list/message_mods)
+	if(HAS_TRAIT(M, TRAIT_SIGN_LANG)) //Forces Sign Language users to wear the translation gloves to speak over radios
+		var/mob/living/carbon/mute = M
+		if(istype(mute))
+			var/empty_indexes = mute.get_empty_held_indexes() //How many hands the player has empty
+			var/obj/item/clothing/gloves/radio/G = mute.get_item_by_slot(ITEM_SLOT_GLOVES)
+			if(!istype(G))
+				return FALSE
+			else
+				if(length(empty_indexes) == 1)
+					message = stars(message)
+				if(length(empty_indexes) == 0) //Due to the requirement of gloves, the arm check for normal speech would be redundant here.
+					return FALSE
```

Comment:
Seems that it would work fine as long as you have at least two hands, and partially if you have one. Maybe it could require specifically at least a left and right hand, but it seems unnecessary.

Reply:
If there's ways to get more than two hands, I didn't know about them. Furthermore, this is the radio code, so it _does_ require both hands due to the fact that you require gloves to speak over the radio. Without both hands, the gloves will fall off, and you can't talk over the radio anymore.

Reply:
Aight that works for me